insecure-ssh/insecure-ssh.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   containers:
     - name: ssh-proxy
-      image: entlein/ssh-proxy:0.0.3
+      image: jamescallaghan1987/insecure_ssh:latest
```

Comment:
question: I changed the name of the image to something inocuous. Is there a difference in the images ? If yes, could you push an image that has a "neutral" name, please?

Reply:
Ah....this was just left over from testing. I changed the image to minimise the number of variables but forgot to change back. Will resolve now :)